### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vaimeo1**

### CODE REVIEW

**Comment by vaimeo** [comment]
It has some error on code

**Comment by vaimeo** [comment]
Please fix the code ASAP

**Comment by vaimeo1** [comment]
> Please fix the code ASAP

Ok

**Review by vaimeo1 — COMMENTED** [review]
Fix it again

**Review by vaimeo — CHANGES_REQUESTED** [review]
Fix it withname

File: README.md [hunk 1]
```diff
@@ -8,3 +8,6 @@ add source and target database
 # 3 Run 
 open the URL yoururl.com/ci-database-diff-generator/
 
+#PR Submit from vaimeo1
+
+#PR Resolved vaimeo1
```
Comment (vaimeo) [inline README.md:13]: Thanks, looks good.

**Review by vaimeo — APPROVED** [review]

**Pull request merged**